### DISCUSSION TITLE
Does using options={     "face_plugins": "calculator"}) lead to additional overhead?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#833] [Q&A] [answered]
Does using options={     "face_plugins": "calculator"}) lead to additional overhead?

**Comment by pospielov** [comment, marked as answer]
If you use a face detection service, then yes, we will have to run one more neural network to calculate the embedding.
If you use a face recognition service, then no, the plugin is used by default. The only difference is that with the plugin enabled we return more information.
